py/makeqstrdefs.py

```diff
@@ -19,6 +20,35 @@
 # Extract MP_COMPRESSED_ROM_TEXT("") macros.  (Which come from MP_ERROR_TEXT)
 _MODE_COMPRESS = "compress"
 
+# File types which will be scanned.
+_CPPFILE_TYPES = [".cc", ".cpp", ".cxx"]
```

Comment:
this should probably be `_CXXFILE_TYPES`

Reply:
Yes. Although in hindsight, I'm not sure it is really needed to have all of these listed?

Reply:
True... it only supports `.cpp` files in the Makefile.  So could just get rid of this variable altogether.  Up to you.

Reply:
Removed it to keep things simple. Note I also removed the `.lower()` on the extension since that is not really correct anyway.